setup.py

```diff
@@ -63,15 +49,15 @@ def is_requirement(line):
     version='0.2.1',
```

Comment:
This generally looks good. @jikamens can you bump this version as well so we can do a new release?